src/transaction.js

```diff
@@ -288,13 +320,27 @@ export class Transaction {
           new xdr.TransactionV1Envelope({ tx, signatures })
         );
         break;
+      case xdr.EnvelopeType.envelopeTypeTxFeeBump():
+        envelope = new xdr.TransactionEnvelope.envelopeTypeTxFeeBump(
+          new xdr.FeeBumpTransactionEnvelope({ tx, signatures })
+        );
+        break;
       default:
-        throw new Error('Invalid transaction');
+        throw new Error(`Invalid EnvelopeType: ${this.envelopeType}.`);
```

Comment:
should `this._envelopeType` be used instead of `this.envelopeType`? I don't think `this.envelopeType` is defined anywhere

Reply:
👍